Assets/Scripts/StandardSamples/Discord/DiscordManager.cs

```diff
@@ -202,6 +248,32 @@ private async Task<TokenReader> GetTokenAsync()
             }
 
             //use received auth code to request auth token
+            const string grant_type = "grant_type";
```

Comment:
Should be `camelCase`. From context, I can see that you got the variable name from the altered code, wherein the original value was `"grant_type"`. I still believe it should be changed, but I can see how the name was determined.

Reply:
Resolved by changing as you suggested. I made this "grantTypeString". Do you think this name doesn't need the -String suffix? I want to make sure its purpose is clear, to have one fewer magic string.

Reply:
I like the practice of having one fewer magic strings. I think it could go either way. I think there is some conflict with code style guideline stuff to do this - but in these cases I will opt to use the `Str` suffix as shorthand, just to indicate that it's a string without adding the full name. Could really go either way. Certainly `String` as a suffix is clearer.